public/react-chat/app.jsx

```diff
@@ -1,5 +1,5 @@
 // A placeholder image if the user does not have one
-const PLACEHOLDER = '/placeholder.png';
+const PLACEHOLDER = 'https://placeimg.com/60/60/people';
```

Comment:
why change this? Now we are entirely dependent on another service just for a plain avatar that is super small.